recipes-kernel/accton-csp7551-mods/accton-csp7551-mods.bb

```diff
@@ -0,0 +1,19 @@
+SUMMARY = "CSP 7551 platform modules"
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://COPYING;md5=12f884d2ae1ff87c09e5b7ccc2c4ca7e"
+
+inherit module
+
+SRC_URI = " \
+          file://Makefile \
+          file://fpga_driver.c \
+          file://accton_i2c_cpld.c \
+          file://x86-64-accton-csp7551-sfp.c \
+          file://COPYING \
+          "
```

Comment:
Please either do a base files + our modifications split as in https://github.com/bisdn/meta-ofdpa/tree/main/recipes-kernel/openbcm-gpl-modules or directly checkout from git (+ our modifications split).

Directly modifying the files makes it harder to do atomic updates of them in case accton releases new versions, and makes it harder to reapply our changes.

Also in case you use the bcm-gpl-modules approach, add a comment where you've taken the files from.

Reply:
Done.

Reply:
> add a comment where you've taken the files from.

I meant that literally, add a comment to the recipe please ;P

Reply:
Done.